src/components/ChannelList/ChannelList.tsx

```diff
@@ -183,8 +216,23 @@ const UnMemoizedChannelList = <
     navOpen = false,
     setActiveChannel,
     theme,
+    themeVersion,
     useImageFlagEmojisOnWindows,
   } = useChatContext<StreamChatGenerics>('ChannelList');
+  console.log('active', channel);
```

Comment:
```suggestion
```

Reply:
thank you @arnautov-anton , I have removed the console.log call